pages/linux/kwriteconfig5.md

```diff
@@ -0,0 +1,32 @@
+# kwriteconfig5
+
+> Write KConfig entries for KDE Plasma.
+> More information: <https://userbase.kde.org/KDE_System_Administration/Configuration_Files>.
+
+- Display help:
+
+`kwriteconfig5 --help`
+
+- Set a global configuration key:
+
+`kwriteconfig5 --group {{group_name}} --key {{key_name}} {{value}}`
+
+- Set a key in a specific configuration file:
+
+`kwriteconfig5 --file {{path/to/file}} --group {{group_name}} --key {{key_name}} {{value}}`
+
+- Delete a key:
+
+`kwriteconfig5 --group {{group_name}} --key {{key_name}} --delete`
+
+- Use systemd to start the Plasma session when available:
+
+`kwriteconfig5 --file {{startkderc}} --group {{General}} --key {{systemdBoot}} {{true}}`
+
+- Hide the title bar when a window is maximized (like Ubuntu):
+
+`kwriteconfig5 --file {{~/.config/kwinrc}} --group {{Windows}} --key {{BorderlessMaximizedWindows}} {{true}}`
+
+- Open KRunner with the Meta (Command/Windows) key:
```

Comment:
What about this?
```suggestion
- Configure KRunner to open with the Meta (Command/Windows) global hotkey:
```

Reply:
Yep exactly.

Reply:
:thinking: As a user you're configuring the Plasma Shell to open KRunner with the Meta key.